src/processElement.js

```diff
@@ -1,82 +1,75 @@
 import * as  cheerio from 'cheerio';
 
-export const processElement = element => {
+export const processElement = element => new Promise((resolve, reject) => {
```

Comment:
Same here. Just return a promise. No need for `new Promise` and `resolve` and `reject`.